editor/src/templates/index.html

```diff
@@ -25,6 +25,18 @@
       crossorigin="anonymous"
     ></script>
 
+    <% if (VITE) { %>
+    <script>
+      window.global = window
+      window.require = () => {}
+      window.exports = {}
```

Comment:
I'm somewhat terrified that this is needed. clearly this prevents code that uses exports from _throwing_ an error. but it will still not actually export anything!

Reply:
Hmmm I wonder if actually some of these can go in the bin now. Will verify and comment

Reply:
These have now been removed (along with the Buffer script)